pkg/blueprint/blueprint.go

```diff
@@ -29,10 +29,12 @@ type Group struct {
 }
 
 type Container struct {
-	Source string `json:"source" toml:"source"`
-	Name   string `json:"name,omitempty" toml:"name,omitempty"`
+	Source string  `json:"source,omitempty" toml:"source"`
+	Name   string  `json:"name,omitempty" toml:"name,omitempty"`
+	Digest *string `json:"digest,omitempty" toml:"digest,omitempty"`
 
-	TLSVerify *bool `json:"tls-verify,omitempty" toml:"tls-verify,omitempty"`
+	TLSVerify         *bool `json:"tls-verify,omitempty" toml:"tls-verify,omitempty"`
+	ContainersStorage *bool `json:"containers-storage,omitempty" toml:"containers-storage,omitempty"`
```

Comment:
Just went with this as a key rather than something like `local` or `local-container`... but one of those might be clearer.